client/src/store/index.ts

```diff
@@ -283,7 +283,8 @@ const initState = {
   me: null,
   allUsers: [],
   reviewMode: false,
-  currentProject: null as Project | null,
+  globalSettings: undefined as ProjectSettings,
+  currentProject: undefined as Project | null,
```

Comment:
For my own understanding, `currentProject == undefined` means no project is selected, `currentProject == null` means the global project is selected, otherwise `currentProject` is the selected project?

Reply:
Correct.